userial/ad26.c

```diff
@@ -251,10 +251,17 @@ float Volt_Reading(int portnum, int vdd, int *cad)
 	    if(cad) {
 
 	      /* Get Current reading as well */
+              /* convert from 12-bit 2's complement, see
```

Comment:
It's actually 10 bit. The current register is 10 bits, the offset register (which is used internally to update the current register) is 12 bits. Should probably reference the data sheet table 3 here instead of wikipedia.